readme.md

```diff
@@ -45,6 +45,22 @@ echo Carbon::now()->subMinutes(2)->diffForHumans(); // '2 minutes ago'
 // rolling up to seconds, minutes, hours, days, months, years
 
 $daysSinceEpoch = Carbon::createFromTimestamp(0)->diffInDays();
+
+// Check to see if a date is a business day defaulting to a Monday
+// through Friday work week
+$today = Carbon::now();
+CarbonExtensions::isBusinessDay($today);
```

Comment:
isWeekday is not the same as isBusinessDay.  Weekdays are Monday-Friday whereas a business might only be open Friday-Sunday. Additionally, isBusinessDay checks to see if the day is a holiday if a holiday schedule is passed in as a parameter.

For example: A business operates on a Monday - Friday schedule. The current date is on a Friday and the following Monday is a holiday. You need to know what the date is going to be for 3 business days from now. Since the following Monday is a holiday the date would be for the following Thursday.

While the basic functionality of isBusinessDay does the same as isWeekday, the difference is that it allows for custom business and holiday schedules.  If you take a look at the unit tests you'll find examples of how this is used.